### DISCUSSION TITLE
Testing Issues w/ Windows

### DISCUSSION THREAD

**Discussion by Caleb-Cohen** [Together-100Devs/Together#399] [General]
Can individuals please post the types of errors that are occurring on windows with tests, prettier, and lint please?

**Comment by itisMilan** [comment]
This was the issue that I was getting before changing  "endOfLine" in .prettierc from lf to auto.
```
11:4   error  Delete `␍`  prettier/prettier
  12:65  error  Delete `␍`  prettier/prettier
  13:36  error  Delete `␍`  prettier/prettier  14:25  error  Delete `␍`  prettier/prettier  15:16  error  Delete `␍`  prettier/prettier
  16:3   error  Delete `␍`  prettier/prettier
  17:1   error  Delete `␍`  prettier/prettier
  18:28  error  Delete `␍`  prettier/prettier

✖ 5327 problems (5327 errors, 0 warnings)
  5327 errors and 0 warnings potentially fixable with the `--fix` option.  

```

**Comment by sylfort** [comment]
I didn’t get any errors when running ‘npm run lint’ but when I run ‘npm run test’ I get a lot of errors and the test get stuck in the 2 out of 5 part. Here are the logs:

https://pastebin.com/NU3TTaDk

**Comment by soph-em** [comment]
I've not had issues since changing endOfLine to auto, I had the same issue as Milan before.
